src/pudl/output/sec10k.py

```diff
@@ -106,3 +110,135 @@ def out_sec10k__changelog_company_name(
         name_changelog["company_name_old"] != name_changelog["company_name_new"]
     ]
     return name_changelog
+
+
+def _get_n_duplicate_filer_records(company_info_df, ownership_df, key):
+    filers_count_df = (
+        company_info_df.groupby(key).size().reset_index(name="filer_count")
+    )
+    non_unique_filers_df = filers_count_df[filers_count_df.filer_count > 1]
+    return len(
+        ownership_df.merge(
+            non_unique_filers_df,
+            on=key,
+            how="inner",
+        ).drop_duplicates(subset=key)
+    )
+
+
+@dg.asset(
+    io_manager_key="pudl_io_manager",
+    group_name="out_sec10k",
+)
+def out_sec10k__parents_and_subsidiaries(
```

Comment:
Should this table be sorted by `report_date` and `filename_sec10k`? Is there a standard here?